src/inatinqperf/adaptors/qdrant_adaptor.py

```diff
@@ -12,6 +16,38 @@
 from inatinqperf.adaptors.base import DataPoint, HuggingFaceDataset, Query, SearchResult, VectorDatabase
 from inatinqperf.adaptors.enums import Metric
 
+HTTP_STATUS_OK = 200
+DEFAULT_GRPC_PORT = 6334
+_ALLOWED_CONTAINER_CHARS = set("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.")
+
+
+def _is_safe_container_name(name: str) -> bool:
```

Comment:
Can we move these to another file or module since they are not specific to the qdrant adaptor?